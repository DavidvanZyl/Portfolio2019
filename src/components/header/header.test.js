import React from "react"
import Header from "./header"

describe("Header", () => {
  test("renders correctly", () => {
    const wrapper = shallow(<Header />)
    expect(wrapper).toMatchSnapshot()
  })
})
