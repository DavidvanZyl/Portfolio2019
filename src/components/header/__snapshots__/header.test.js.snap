// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="header"
>
  <div
    className="header__content"
  >
    <p
      style={
        Object {
          "margin": 0,
        }
      }
    >
      <ForwardRef
        className="link"
        to="/"
      />
    </p>
    <p
      style={
        Object {
          "margin": 0,
        }
      }
    >
      <ForwardRef
        className="link"
        to="/"
      >
        Download my CV
      </ForwardRef>
    </p>
  </div>
</header>
`;
